package.json

```diff
@@ -37,7 +37,7 @@
         "!artifacts/contracts/interfaces/**/*.dbg.json"
     ],
     "scripts": {
-        "contracts:run": "hardhat compile && hardhat deploy",
+        "contracts:run": "hardhat compile --force && hardhat deploy",
```

Comment:
Would it be beneficial to also have `npx hardhat clean` in this command?

Reply:
I agree. With constant branch switching, I often run into problems and have to run` npx hardhat clean`. It's imperative to do `npx hardhat clean` and `npx hardhat compile --force before` a deploy. Both should be added to all of the `contracts:migrate` npm scripts

Reply:
done

Reply:
OK for me.